### ISSUE TITLE
Ludolf's Historia and Commentarius

### ISSUE THREAD

**Issue by PietroLiuzzo** [BetaMasaheft/Documentation#1702]
@Giuliacas is working on encoding Ludolf's [Commentarius](https://archive.org/details/bub_gb_VOtTAAAAcAAJ/page/n41/mode/2up) within her Erasmus Plus trainship from the University of Bologna.

I think that, although this is probably not a work of Ethiopian Literature, it is relevant in our research environment as other similar resources and it is strictly related to the scope of the project.

Since I did not find them, we would like to create
- a Work record for the Commentarius
- a Work record for the Historia

Both will contain a transcription done in Transkribus for the Latin text, which is being corrected by @Giuliacas. 

In the case of the _Commentarius_ the text, beside being fully referenceable will also be enriched with 

- formal references to the Historia, and 
- named entities annotations:
     - personal names pointing to BM entities
     - toponyms pointing to BM entities
     - references to works and manuscripts (?) pointing to BM entities
     - dates
     - bibliographical references pointing to the common Zotero library
     - foreign languages 
     - quotations

The _Historia_ will also contain the transkribed (but not segmented) text of the available related resources: the French and English translations, the Appendixes, with references to the images available online.

Please @thea-m @DenisNosnitsin1970 @eu-genia @DariaElagina let us know if you think this should not be done this way. The alternative is that this work is published as a separate application (pointing and using BM).

**Comment by DenisNosnitsin1970** [comment]
I personally have nothing against full inclusion, but I think that this is a different type of material as compared to the main subject of Beta masaheft, and represent quite a digression. There are more works of comparable importance compiled by Europeans (though Ludolf's work may be considered unique), that will possibly be encoded in the future. Because of this, a separate application related to BM would be perhaps more preferable, in my opinion.

**Comment by eu-genia** [comment]
I agree with Denis that there is a problem; 
work is a work of Ethiopian literature, we even cancelled a number of records for works in other oriental literatures that only exist in Ethiopic as chapters or quotations, even if these are real indigenous works
Ludolf's texts are early studies, they cannot be treated as Works in our definition. So yes I would be for a separate app - or at least a separate type of record (which may or may not have the same structure as a Work record) - for digitized secondary sources for Ethiopian and Eritrean studies

**Comment by PietroLiuzzo** [comment]
@abausi @SusanneHummel @smaugustine @MassimoVilla @antobrita perhaps you want to share your ideas as well on this topic? I do not think we already have similar cases and we could document this decision for future similar cases. 
Of course searching will not be possible across separate apps, while if these texts are included, you will find occurrences of a given name or person also in these works and viceversa. A different type may be a solution to distinguish and at the same time have it fully included into the VRE, but I am not sure the earliness helps to make a distinction here, neither the 'european/ethiopian' opposition. what type would this be? and what would this new type include?

**Comment by eu-genia** [comment]
I would call the type "study", it does not matter Ethiopian or European - it would include secondary sources on Ethiopia and Eritrea, scholarly elaborations of any age or type - the full "unedited" text of Dillmann's Lexicon as is could also be part of this category

**Comment by DenisNosnitsin1970** [comment]
I agree with Eugenia, yes, smth like "study"-category.

**Comment by thea-m** [comment]
Thank you for the interesting question!

I agree with @eu-genia  and @DenisNosnitsin1970 , for the following reasons:

- Whether this work will be published on BM or a separate application is, for me (and I can imagine for users not interested the technical infrastructure) of secondary importance, access and use might be more straightforward if everything is found in the same place.
- Creating a normal work record and thus assigning CAe identifiers seems to me very problematic, for the reasons outlined by @eu-genia above. If we would do this - where do we draw the line then? Of course, Ludolf's works are of exceptional and outstanding historical importance, but I cannot come up now with an argument why they should receive a CAe identifier and for instance Antonella's latest article shouldn't. (A recent deletion referenced by @eu-genia was https://github.com/BetaMasaheft/Documentation/issues/1684)
- "Study" as a category could work, or even something broader as "further related resources"

Wherever I will have the pleasure to read these annotated texts, I look forward to it very much! 📚

**Comment by eu-genia** [comment]
(deleting: I was rather referring to cases like https://github.com/BetaMasaheft/Documentation/issues/1617 - but if we create a wider type, something like "sources on Ethiopia/Eritrea", then actually this particular deletion could also be restored and moved to that new type)

**Comment by PietroLiuzzo** [comment]
This all make sense, of course. 
It does have a huge impact on the entire infrastructure, but that is of secondary concern and it is a good time to introduce this, since the backend is being redesigned.
 
I think we need to draw that line precisely and document it precisely in the GL.
 
We have an example or two here, we need a clear definition of what 'studies' are with more examples to help ourself out of future doubts. 

Any one would like to contribute a clear and concise formulation which makes clear the distinction out of any doubt possibly based on as few criteria as possible?

Thanks!

**Comment by PietroLiuzzo** [comment]
I do not want to roll out already the separate application, if anyone likes that idea more, ways to make things interact abound, and the Dillmann example fits well here.

**Comment by eu-genia** [comment]
I am rather against the separate app, - my suggestion was a new type - but we must of course decide if only for Studies or also for other Sources on Eth/Er History/Literature etc.

And - whether Study or Source - my suggestion was that a record should then also be created for DillmLex

**Comment by thea-m** [comment]
Can "attestation in an Ethiopian/Eritrean manuscript" work as a criterion for creating work records/assigning CAe identifiers (apart from the known criteria for the creation of work records as documented in the guidelines https://betamasaheft.eu/Guidelines/?id=works)? I believe that this is what has been followed intuitively and as there is a certain agreement in this present issue discussion thread, I don't see a need for more conceptual work (obviously, please correct me). 
Similarily, "Works of interest to Ethiopian and Eritrean studies" as a criterion for the studies/resources category would from my point of view be sufficient. I see no disadvantage in keeping this definition very open.

There will be interesting border cases of (European) printed editions which were copied in manuscripts and thus integrated into the manuscript culture, which can maybe be discussed when someone will be working on such a case.

(sorry, @PietroLiuzzo , I don't understand your last comment. I agree with @eu-genia 's answer though :) )

**Comment by PietroLiuzzo** [comment]
Dear @thea-m my last comment was meant to leave the discussion open to the many other participants pinged. That is to say, although for now there seems to be some convergence towards a new 'type' for 'studies' within BM to host things like the _Commentarius_ without giving them a CAe, others may prefer a separate application, or argue for the inclusion in the Clavis or come up with another idea not yet considered here, and I wanted to make sure that they did not feel like this was already over and the other options already rolled out. I was simply trying to make sure that the speed of the conversation did not suffocate the discussion in an attempt to welcome other options and opinions, which I hereby reiterate.  
The attestation in manuscripts is a simple and clear criterion to add to the definition IMHO, even better if it follows intuition.
I am also ok with the open definition for 'studies'.

**Comment by abausi** [comment]
I also agree with the general conclusions: 1) no separate app, 2) new category, 3) new label ("studies", "secondary sources" etc.). Yet, there might be cases where printed texts which occur in these "studies" (for example, in the "Commentarius") or whatever we want to call them, are real works and can receive a CAe number. CAe identify Ethiopic works, so that it does not really matter if they are attested by prints or manuscripts. Still, all in all, these cases will be marginal, but we will have some of them. In the case of Dillmann, the idea behind the attribution of a specific identifier to his _Lexicon_ as such was the need to make accessible the dictionary, exactly in the form as Dillmann formulated it, without any alteration, but nobody thought it should receive a CAe entry.

**Comment by PietroLiuzzo** [comment]
Thanks for the contribution, I think I will leave this anyway open until next week, in case someone has second thoughts or those who did not have a say want to share their view. if nothing else is brought forward I will procede with the necessary changes to the infrastructure first, but leave the update of the guidelines for after the release of the application. We will use the _Commentarius_ and _Historia_ as test cases.

**Comment by PietroLiuzzo** [comment]
https://github.com/BetaMasaheft/Studies

**Comment by PietroLiuzzo** [comment]
https://github.com/BetaMasaheft/BetMas/commit/1f4464280a9283d672f806d04a053547448264ef
